Gemfile-rails-dependencies

```diff
@@ -17,9 +17,9 @@ when /stable$/
 when nil, false, ""
   if RUBY_VERSION < '1.9.3'
     # Rails 4+ requires 1.9.3+, so on earlier versions default to the last 3.x release.
-    gem "rails", "~> 3.2.17"
+    gem "rails", "~> 3.2"
```

Comment:
This are all development dependencies. I consider a good thing that contributors try out newest versions and find out about breakages.